YAXLib/Attributes/YAXCustomSerializerAttribute.cs

```diff
@@ -34,14 +34,14 @@ public YAXCustomSerializerAttribute(Type customSerializerType)
         void IYaxMemberLevelAttribute.Setup(MemberWrapper memberWrapper)
         {
             EnsureDesiredInterface(memberWrapper.MemberType);
-            memberWrapper.CustomSerializerType = CustomSerializerType;
+            memberWrapper.CustomSerializer = new CustomSerializerWrapper(CustomSerializerType);
```

Comment:
The `MemberWrapper` stores an instance of the custom serializer. Before, a knew instance was created per invokation of custom serializer.